spec/models/rapids/work_process_spec.rb

```diff
@@ -16,5 +16,37 @@
       expect(work_process.minimum_hours).to eq 20
       expect(work_process.maximum_hours).to eq 200
     end
+
+    it "returns work process with competencies if available" do
+      work_process_response = create(
+        :rapids_api_detailed_work_activity,
+        tasks: ["Competency #1; Competency #2; Competency #3"]
```

Comment:
Looks good! Can you remove a space in one of these to check the new regex?
```suggestion
        tasks: ["Competency #1; Competency #2;Competency #3"]
```